image/asif/asif_darwin_test.go

```diff
@@ -0,0 +1,77 @@
+package asif
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+)
+
+func TestOpenASIF(t *testing.T) {
+	// Check macOS version
+	if productVersion, err := exec.CommandContext(context.Background(), "sw_vers", "--productVersion").Output(); err != nil {
```

Comment:
t.Context should be used.
Same for other commands too, and please squash the commits